packages/react-composites/src/composites/CallComposite/pages/LobbyPage.tsx

```diff
@@ -36,6 +41,17 @@ export const LobbyPage = (props: LobbyPageProps): JSX.Element => {
   const callState = useSelector(getCallStatus);
   const inLobby = callState === 'InLobby';
 
+  /* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */
+  const participants = useSelector(getRemoteParticipants) ?? {};
+
+  const remoteParticipantsTrampoline = (): RemoteParticipantState[] | undefined => {
```

Comment:
Do we need this trampoline?
I thought remoteParticipants are only available for outbound 1:1 or PSTN calls?